File: Kkang'sAndroid/Java/Session08_TestDBMS/app/src/main/java/com/opusone/leanon/session08_testdbms/DBHelper.java
package com.opusone.leanon.session08_testdbms;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

public class DBHelper extends SQLiteOpenHelper {

    public static final int DATABASE_VERSION = 1;

    public DBHelper(Context context){
        super(context, "java_memodb", null, DATABASE_VERSION);
    }

    @Override
    public void onCreate(SQLiteDatabase db){
        String memoSQL = "create table java_memodb " + "(_id integer primary key autoincrement, " +
                "title, " +
                "content)";

        db.execSQL(memoSQL);


    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion){
        db.execSQL("drop table java_memodb");
    }


}

File: Kkang'sAndroid/Kotlin/Session08_TestDBMS/app/src/main/java/com/opusone/leanon/testdbms/DBHelper.kt
package com.opusone.leanon.testdbms

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class DBHelper(context: Context) :
    SQLiteOpenHelper(context, "kotlin_memodb", null, 1) {

    override fun onCreate(db: SQLiteDatabase?) {
        val memoSQL = "create table kotlin_memodb " + "(_id integer primary key autoincrement, " +
                "title, " +
                "content)"

        db?.execSQL(memoSQL)
    }

    override fun onUpgrade(db: SQLiteDatabase?, oldVersion: Int, newVersion: Int) {
        db?.execSQL("drop table kotlin_memodb")
        onCreate(db)
    }
}